test/spec/modules/kargoBidAdapter_spec.js

```diff
@@ -356,4 +356,81 @@ describe('kargo adapter tests', function () {
       expect(resp).to.deep.equal(expectation);
     });
   });
+
+  describe('user sync handler', function() {
+    const clientId = '74c81cbb-7d07-46d9-be9b-68ccb291c949';
+    var uid;
+    var shouldSimulateOutdatedBrowser;
+
+    beforeEach(() => {
+      uid = {};
+      shouldSimulateOutdatedBrowser = false;
+
+      sandbox.stub(crypto, 'getRandomValues').callsFake(function(buf) {
```

Comment:
This currently returns an `undefined` error on the `crypto` object when the unit tests run in IE11 (see the circleci failed job for details).  Can you please take a look and fix the issue?